src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,202 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Parser class allows the Duke program
+ * to parse user inputs into commands.
+ *
+ * @author Gerald Teo Jin Wei
+ * @version 0.1
+ * @since 2022-08-28
+ */
+public class Parser {
+    public Parser() {}

```

Comment:
This seems to be dead code. One way to perhaps make it somewhat useful would be to write it as
`private Parser() {}`, to make it such that the class cannot be instantiated.